scopes/workspace/workspace/pick-outdated-pkgs.ts

```diff
@@ -0,0 +1,130 @@
+import colorizeSemverDiff from '@pnpm/colorize-semver-diff';
+import semverDiff from '@pnpm/semver-diff';
+import { OutdatedPkg } from '@teambit/dependency-resolver';
+import { getBorderCharacters, table } from '@zkochan/table';
```

Comment:
@zkochan Why do we need another table?
we already have both `table` and `cli-table` (and maybe even `tty-table`)

Reply:
We may probably use `table` instead. The only reason I use a fork of `table` is that I changed `slice-ansi` to `@arcanis/slice-ansi` in the fork. There is a bug in `slice-ansi` but looks like it doesn't affect us, so I'll use table.